speech/grpc/src/main/java/com/examples/cloud/speech/StreamingRecognizeClient.java

```diff
@@ -76,6 +81,10 @@ public StreamingRecognizeClient(ManagedChannel channel, String file, int samplin
     this.channel = channel;
 
     speechClient = SpeechGrpc.newStub(channel);
+
+    //Send log4j logs to Console
```

Comment:
Shouldn't this be configured in a configuration file, rather than in code? eg if a user wanted to log to a file instead of stdout..